### DISCUSSION TITLE
Esistono funzionalità per per cancellare/minimizzare i dati personali non rilevanti?

### DISCUSSION THREAD

**Discussion by marcomeneghin** [globaleaks/globaleaks-whistleblowing-software#4561] [Q&A] [answered]
Un Organismo di Vigilanza chiede che la piattaforma preveda la possibilità per il Ricevente di cancellare dati personali non pertinenti (è un tema legato alla minimizzazione). Quanto inviato dal segnalante (testi/file) ho visto che è inalterabile. C'è qualche modo (oltre alla possibilità di cancellare in toto una segnalazione) di permettere la cencellazione di dati non pertinenti?

**Comment by evilaliv3** [comment, marked as answer]
Salve Marco,

si, ci sono due funzionalità pensate specificatamente per questo scopo:
- dare ad alcuni utenti la possibilità di mascherare dati non rilevanti: https://github.com/globaleaks/GlobaLeaks/issues/3420
- dare ad alcuni utenti la possibilità di cancellare indelebilmente dati non rilevanti: https://github.com/globaleaks/GlobaLeaks/issues/3429
